backend_new/src/services/application-csv-export.service.ts

```diff
@@ -0,0 +1,653 @@
+        },
+        {
+          path: `householdMember.${i}.birthYear`,
+          label: `Household Member (${j}) Birth Year`,
+        },
+        {
+          path: `householdMember.${i}.sameAddress`,
+          label: `Household Member (${j}) Same as Primary Applicant`,
+        },
+        {
+          path: `householdMember.${i}.relationship`,
+          label: `Household Member (${j}) Relationship`,
+        },
+        {
+          path: `householdMember.${i}.workInRegion`,
+          label: `Household Member (${j}) Work in Region`,
+        },
+        {
+          path: `householdMember.${i}.street`,
+          label: `Household Member (${j}) Street`,
+        },
+        {
+          path: `householdMember.${i}.street2`,
+          label: `Household Member (${j}) Street 2`,
+        },
+        {
+          path: `householdMember.${i}.city`,
+          label: `Household Member (${j}) City`,
+        },
+        {
+          path: `householdMember.${i}.state`,
+          label: `Household Member (${j}) State`,
+        },
+        {
+          path: `householdMember.${i}.zipCode`,
+          label: `Household Member (${j}) Zip Code`,
+        },
+      );
+    }
+
+    return headers;
+  }
+
+  async getCsvHeaders(
+    maxHouseholdMembers: number,
+    multiSelectQuestions: MultiselectQuestion[],
+    includeDemographics = false,
+  ): Promise<CsvHeader[]> {
+    const headers: CsvHeader[] = [
+      {
+        path: 'id',
+        label: 'Application Id',
+      },
+      {
+        path: 'confirmationCode',
+        label: 'Application Confirmation Code',
+      },
+      {
+        path: 'submissionType',
+        label: 'Application Type',
+      },
+      {
+        path: 'submissionDate',
+        label: 'Application Submission Date',
+      },
+      {
+        path: 'applicant.firstName',
+        label: 'Primary Applicant First Name',
+      },
+      {
+        path: 'applicant.middleName',
+        label: 'Primary Applicant Middle Name',
+      },
+      {
+        path: 'applicant.lastName',
+        label: 'Primary Applicant Last Name',
+      },
+      {
+        path: 'applicant.birthDay',
+        label: 'Primary Applicant Birth Day',
+      },
+      {
+        path: 'applicant.birthMonth',
+        label: 'Primary Applicant Birth Month',
+      },
+      {
+        path: 'applicant.birthYear',
+        label: 'Primary Applicant Birth Year',
+      },
+      {
+        path: 'applicant.emailAddress',
+        label: 'Primary Applicant Email Address',
+      },
+      {
+        path: 'applicant.phoneNumber',
+        label: 'Primary Applicant Phone Number',
+      },
+      {
+        path: 'applicant.phoneNumberType',
+        label: 'Primary Applicant Phone Type',
+      },
+      {
+        path: 'additionalPhoneNumber',
+        label: 'Primary Applicant Additional Phone Number',
+      },
+      {
+        path: 'contactPreferences',
+        label: 'Primary Applicant Preferred Contact Type',
```

Comment:
If there are more than one selected "preferred contact type" it pushes everything over. This looks like it is due to the output should have commas and CSV interprets commas as a new column. 
<img width="923" alt="image" src="https://github.com/bloom-housing/bloom/assets/42942267/9c621666-9215-4f29-b2c4-e154815a4811">